src/python/tests/appengine/handlers/cron/grouper_test.py

```diff
@@ -249,3 +251,30 @@ def test_different_project_name_with_similar_state(self):
       self.testcases[index] = data_handler.get_testcase_by_id(t.key.id())
       self.assertEqual(self.testcases[index].group_id, 0)
       self.assertTrue(self.testcases[index].is_leader)
+
+
+@test_utils.with_cloud_emulators('datastore')
+class GroupExceedMaxTestcasesTest(unittest.TestCase):
```

Comment:
Different class, since general grouper testcases always create two helper testcases in setUp.